### DISCUSSION TITLE
Change the timeline/trace/report/dag file path within the script

### DISCUSSION THREAD

**Discussion by santiagorevale** [nextflow-io/nextflow#2486] [Q&A] [answered]
Hi there,

I was wondering if it would be possible to set the file path for the timeline/trace/report/dag from within a script.

My pipeline dynamically retrieves some data that will be used to set up the output directory hence why I can't define on the `nextflow.config` file where I want these files to be saved.

Thank you very much in advance.

Cheers,
Santiago

**Comment by pditommaso** [comment, marked as answer]
Nope, it can only be controlled via CLI option or the config file.

**Reply by santiagorevale** [reply]
Thanks, Paolo.
